js/utils.js

```diff
@@ -1140,6 +1140,28 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     });
   }
 
+  function openLinkAction(options) {
+    var entry = _.find(options.records, function(entry) {
+      return entry.id === options.recordId;
+    });
+
+    if (!entry) {
+      return;
+    }
+
+    var value = entry.data[options.summaryLinkAction.column];
```

Comment:
I'd also add tests for `options.summaryLinkAction.column` and `value` so that if `options.summaryLinkAction` isn't valid then `return` and do nothing.

Reply:
@tonytlwu done, please review